doc/api/util.md

```diff
@@ -327,6 +327,8 @@ changes:
 * `options` {Object}
   * `showHidden` {boolean} If `true`, the `object`'s non-enumerable symbols and
     properties will be included in the formatted result. Defaults to `false`.
+  * `tagOnly` {boolean} If `true`, only the tag of `object` will be shown,
```

Comment:
there may need to be additional detail about what is meant by "tag" here.

Reply:
i was thinking the same thing, but i wasn't sure what to write, so i just hoped the example below would be enough

Reply:
```
If `true`, only the object type (tag) will be shown, ignoring all keys and values. Printing is like the same object being empty. Defaults to `false`.
```
Something like that?